src/lib/backpack-api.js

```diff
@@ -73,12 +73,30 @@ const deleteBackpackObject = ({
     });
 });
 
+// Two types of backpack items are not retreivable through storage
+// code, as json and sprite3 as arraybuffer zips.
+const fetchAs = responseType => uri => new Promise((resolve, reject) => {
```

Comment:
Maybe I just need to get used to it if other people have a different feeling about it, but this is too many arrows for me to understand what's going on the first time I read it.  I think it crosses the line of syntax convenience that arrow functions provide and starts treading into confusion, making parsing it a burden again.

I think some line breaks and indentation would help readability, if not using a full-on `function () {}` for the outer function.

Reply:
Yeah, honestly this formatting is just a product of the lint auto fixer. I'll fix.